packages/core/src/schema/models/connectedEvaluations.ts

```diff
@@ -0,0 +1,44 @@
+import { relations } from 'drizzle-orm'
+import { bigint, bigserial, index, unique, uuid } from 'drizzle-orm/pg-core'
+
+import { documentVersions, latitudeSchema } from '..'
+import { timestamps } from '../schemaHelpers'
+import { evaluations } from './evaluations'
+
+export const connectedEvaluations = latitudeSchema.table(
+  'connected_evaluations',
+  {
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
+    documentUuid: uuid('document_uuid').notNull(),
```

Comment:
Had to remove the foreign key from `documentUuid`, as it is not a unique/pk field in the referenced table.